### PR TITLE
Add inotify_simple

### PR DESCRIPTION
**Pull request by mwilson8**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mwilson8** [comment]
@rluria14 maintain this with me?

**Comment by mwilson8** [comment]
@conda-forge/staged-recipes
@conda-forge/help-python
This package is ready for review

File: recipes/inotify_simple/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: fc2c10dd73278a1027d0663f2db51240af5946390f363a154361406ebdddd8dd
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [ win ]
```
Comment (xhochy) [inline recipes/inotify_simple/meta.yaml]: Please remove the spaces around the selector

```suggestion
  skip: true  # [win]
```

File: recipes/inotify_simple/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  
+requirements:
+  host:
+    - pip
+    - python
+    - enum34  # [ py<34 ]
+    
+  run:
+    - python
```
Comment (xhochy) [inline recipes/inotify_simple/meta.yaml]: Is `enum34` no run requirement?

Comment (mwilson8) [reply recipes/inotify_simple/meta.yaml]: yes it should be, thank you.. should I also remove spaces on the selector there as well?

Comment (xhochy) [reply recipes/inotify_simple/meta.yaml]: Yes, please at all occurrences

**Pull request merged**